config/packages/framework.yaml

```diff
@@ -17,7 +17,7 @@ framework:
     http_client:
         default_options:
             headers:
-                'User-Agent': 'Mbin/1.2.1 (+https://%kbin_domain%/agent)'
+                'User-Agent': 'Mbin/1.3.0 (+https://%kbin_domain%/agent)'
```

Comment:
not sure why this file show changes, if you rebased with the latest main branch, I would expect to just show 1.3.0.

Reply:
Is now resolved.